ingestion/src/metadata/ingestion/source/database/datalake.py

```diff
@@ -170,7 +170,7 @@ def get_gcs_files(self, bucket_name, prefix):
     def get_s3_files(self, bucket_name, prefix):
         kwargs = {"Bucket": bucket_name}
         if prefix:
-            kwargs["prefix"] = prefix
+            kwargs["Prefix"] = prefix if prefix.endswith("/") else f"{prefix}{'/'}"
```

Comment:
This could be `else f"{prefix}/"`, no need to add the extra braces around the slash